doc/api/http2.md

```diff
@@ -1730,7 +1731,9 @@ instances.
 ```js
 const http2 = require('http2');
 
-// Create a plain-text HTTP/2 server
+// Create a simple HTTP/2 server
+// This will not work in most browsers,
+// Try http2.createSecureServer instead
```

Comment:
A nit: `// Try` -> `// try` after the comma above?

Reply:
Any thoughts on usage of periods `.` anywhere?

Reply:
Let's see what others think, I am not a native speaker)

Reply:
I added periods.